public/app/features/panel/panellinks/link_srv.ts

```diff
@@ -1,15 +1,42 @@
-import angular from 'angular';
 import _ from 'lodash';
+import { TimeSrv } from 'app/features/dashboard/services/TimeSrv';
+import { TemplateSrv } from 'app/features/templating/template_srv';
+import coreModule from 'app/core/core_module';
 import { appendQueryToUrl, toUrlParams } from 'app/core/utils/url';
-import { PanelDrillDownLink, KeyValue } from '@grafana/ui';
+import { PanelDrillDownLink, KeyValue, ScopedVars, DateTime, dateTime } from '@grafana/ui';
+import { TimeSeriesValue, transformAbsoluteTimeRange } from '@grafana/ui';
 
 const DrilldownLinkBuiltInVars = {
   keepTime: '__urlTimeRange',
   includeVars: '__allVariables',
+  seriesName: '__seriesName',
+  dataPointTs: '__dataPointTs',
```

Comment:
Maybe: `__field_name` to match existing field display:
https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/utils/fieldDisplay.ts#L35

Maybe `__value_time` rather than  `__dataPointTs`?  datapoint is an internal idea to TimeSeries

Reply:
> Maybe: __field_name to match existing field display: 

I'm not convinced to use `field` terminology in the UI, and this variable will be used from UI only. In our docs and communication we are very about series, and it feels sort of wrong to introduce field concept here in quite an advance-user feature.

Reply:
> Maybe __value_time

👍

Reply:
I think we can have __field_name as well at some point, but that is something different we still need series name (Same way we have both in the Gauge/BarGauge title template).

regarding snake_case vs camelCase I guess we use snake_case the most so might need to continue with that